src/core/TSQuery.java

```diff
@@ -95,7 +95,13 @@
   
   /** Whether or not to delete the queried data */
   private boolean delete = false;
-  
+
+  /** Whether or not to wait for the deletion to be completed. */
```

Comment:
`@since 2.2`?

Reply:
This is added after 2.4. I tried to follow the pattern for `@since`, assuming this will be accepted for 2.5.